installing/installing-fips.adoc

```diff
@@ -83,4 +83,6 @@ If you are using Azure File storage, you cannot enable FIPS mode.
 
 To apply `AES CBC` encryption to your etcd data store, follow the xref:../security/encrypting-etcd.adoc#encrypting-etcd[Encrypting etcd data] process after you install your cluster.
 
-If you add {op-system-base} nodes to your cluster, ensure that you enable FIPS mode on the machines before their initial boot. See xref:../machine_management/adding-rhel-compute.adoc#adding-rhel-compute[Adding RHEL compute machines to an {product-title} cluster] and link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/chap-federal_standards_and_regulations#sec-Enabling-FIPS-Mode[Enabling FIPS Mode] in the {op-system-base} 7 documentation.
+If you add {op-system-base} nodes to your cluster, ensure that you enable FIPS mode on the machines before their initial boot. See xref:../machine_management/adding-rhel-compute.adoc#adding-rhel-compute[Adding RHEL compute machines to an {product-title} cluster]
+and link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/chap-federal_standards_and_regulations#sec-Enabling-FIPS-Mode[Enabling FIPS Mode] in the {op-system-base} 7 documentation.
+//https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening#con_federal-information-processing-standard-fips_assembly_installing-a-rhel-8-system-with-fips-mode-enabled
```

Comment:
If the PR is for 4.8 and previous release, then no need to add link for https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening#con_federal-information-processing-standard-fips_assembly_installing-a-rhel-8-system-with-fips-mode-enabled.
This link for how to enable fips for Rhel8 only. It is not used for RHCOS(https://github.com/openshift/openshift-docs/pull/35858/files#diff-7459a6ec2495b72a5d3059b54825d073d32c3d99e8192fb56035e2a502746146R70) or Rhel7.